lib/modules/manager/custom/regex/readme.md

```diff
@@ -4,17 +4,17 @@ The `regex` manager is unique in Renovate because:
 
```

Comment:
Change the `regex` mentions above this but also add a note that "customManagers was formerly known as regexManagers" for anyone searching the docs